libs/libcommon/src/libcommon/croissant_utils.py

```diff
@@ -192,4 +197,18 @@ def feature_to_croissant_field(
             field["isArray"] = True
             field["arrayShape"] = ",".join([str(shape) if shape else "-1" for shape in array_shape])
             return field
+
+    elif Date is not None and isinstance(feature, Date):
```

Comment:
I think we infer this through the `Value` dtype, see the `if isinstance(feature, Value)` condition at the beginning of the function.